src/protocol/requests/listGroups/v0/response.js

```diff
@@ -0,0 +1,41 @@
+const Decoder = require('../../../decoder')
+const { failure, createErrorFromCode } = require('../../../error')
+
+/**
+ * ListGroups Response (Version: 0) => error_code [groups]
+ *   throttle_time_ms => INT32
```

Comment:
I don't think that `throttle_time_ms` is part of v0. I think that was added in V1. The code looks like it's doing the right thing, so maybe this was a copy-paste mistake.